demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -22,52 +22,53 @@ data:
         params:
       - name: default
         impl: denyChecker
-    attributes:
-      - name: source.name
-        value_type: 1 # STRING
-      - name: source.uid
-        value_type: 1 # STRING
-      - name: target.name
-        value_type: 1 # STRING
-      - name: target.uid
-        value_type: 1 # STRING
-      - name: target.service
-        value_type: 1 # STRING
-      - name: response.code
-        value_type: 2 # INT64
-      - name: response.duration
-        value_type: 10 # DURATION
-
-      # DEPRECATED ATTRIBUTES, TO BE REMOVED:
-      - name: response.latency    # Use response.duration
-        value_type: 10 # DURATION
-      - name: response.http.code  # Use response.code
-        value_type: 2 # INT64
-      - name: source.service      # Use source.name
-        value_type: 1 # STRING
-    metrics:
-      - name: request_count
-        kind: 2 # COUNTER
-        value: 2 # INT64
-        description: request count by source, target, service, and code
-        labels:
-        - name: source
+    manifests:
+      - name: istio-proxy
+        revision: "1"
+        attributes:
+        - name: source.name
```

Comment:
Hey, can we make this look better like we're doing for descriptors elsewhere?

attributes:
  source.name: STRING
  response.code: INT64

etc...